### DISCUSSION TITLE
2024 Cohort 2 - Group 19 Discussion

### DISCUSSION THREAD

**Discussion by sharon-digipres** [Digital-Preservation-Coalition/PythonStudyGroups#45] [General]
This is the discussion thread for those in **Group 19** of the July-December 2024 Cohort

**Comment by asegal20902** [comment]
Hi all Group 19 members! I am going to repost the material I posted during the last cohort, where I was not as active. I'd like to start things off with a slide I produced during the pandemic on how I used Python at the Library of Congress during a learning group to use the Python Library "Beautiful Soup" to retrieve Library data about the papers of President William Howard Taft, which I helped get online, and about a book of poetry by great weird fiction author Clark Ashton Smith, because I am interested in science fiction and fantasy ;) . I would also like to share the link from Digital Transitions, a camera company which produces imaging cameras and software that the Library of Congress uses, to a webinar about how to use Python. https://www.youtube.com/watch?v=mozYgcGzjqs
[Showcase-Show-Tell.pptx](https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/files/14231178/Showcase-Show-Tell.pptx)
Perhaps we can discuss more of what we are doing on Friday mornings when I am telecommuting. 
Best Ariel Segal

**Comment by asegal20902** [comment]
[Showcase-Show-Tell.pptx](https://github.com/user-attachments/files/16698008/Showcase-Show-Tell.pptx)
Here's the slide on what I did for the Library of Congress learning group. As I am rusty, I might want to run through it again. This includes some presentations by other participants as well.

**Comment by asegal20902** [comment]
By the end of Labor Day weekend (beginning of first week in September), I am planning to

Read through the primer https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/wiki
Review the Digital Transitions Webinar on Python using Visual Studio Code
Read through the "Working with APIs" use case https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/blob/main/How-Tos/Use%20Case%20-%20Working%20with%20apis.md and try to make it work using Visual Studio Code.
Does that sound reasonable for the rest of us to do? Any other suggestions?

**Comment by asegal20902** [comment]
Here's a link to the examples and images the Digital Transitions webinar staff provided, available on my Google Drive. https://drive.google.com/drive/folders/149MkvZd-JgM6O2aqK1WjgkokhooHlXZH?usp=sharing
